profile/README.md

```diff
@@ -1,3 +1,28 @@
-# Welcome in the Sumo Logic organization
+# Hi, we are Sumo Logic
 
-Home for the Sumo Logic supported projects that help you work with us.
+We help companies and individual developers to reduce downtime with `real-time alerting`, `dashboards`, and `machine-learning-powered analytics` for all three types of telemetry — `logs, metrics, and traces`.
```

Comment:
A nice photograph or at least a logo would make this page feel a bit more alive ;)

Reply:
Yeah, that's a good idea - there is a log already at the top of this page (it's included in the main org page).
But I can look for a photo on our main site.

Reply:
It looks something like that already:
![image](https://user-images.githubusercontent.com/58700054/134681135-d7aa0f09-d59b-45d0-a5fc-5637f1c3398a.png)